lib/frequency_table.rb

```diff
@@ -6,33 +6,51 @@
 require "overlap/ht_item_overlap"
 
 class FrequencyTable
-  def initialize
-    @table = Hash.new do |hash, member|
-      hash[member] = Hash.new { |fmt_hash, fmt| fmt_hash[fmt] = Hash.new(0) }
+  protected attr_reader :table
+
+  def initialize(data: nil)
+    @table = case data
+    when String
+      JSON.parse(data, symbolize_names: true)
+    when Hash
+      deep_copy data
+    when NilClass
+      {}
+    else
+      raise "unrecognized data format #{data.inspect}"
     end
   end
 
-  def organizations
-    @table.keys.sort
+  def to_json
+    JSON.generate(table)
+  end
+
+  def ==(other)
+    self.class == other.class && table == other.table
   end
 
-  def [](organization)
-    @table[organization.to_sym]
+  def organizations
```

Comment:
Is `organizations` used outside of tests?
If it's only used in tests I'd remove it and replace with `ft.table.keys.sort` or similar where needed.